srcpkgs/xrdb/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'xrdb'
 pkgname=xrdb
-version=1.2.0
-revision=2
+version=1.2.1
+revision=1
 build_style=gnu-configure
 configure_args="--with-cpp=/usr/bin/mcpp"
```

Comment:
```suggestion
configure_args="--with-cpp=/usr/bin/cpp,/usr/bin/mcpp"
```

Reply:
I like your suggestion. Idk if it works, but `clang-cpp` could also be added if you want to cover everything.

Reply:
I like the idea of including clang-cpp.  I did some minimal testing with some `#include` and `#ifdef` directives, and made sure the `-I` and `-D` flags work, but as far as I can tell clang-cpp works too.  I have cpp preferred first, since it is probably the expected default, then clang-cpp, and then use mcpp as fallback.